javascripts/discourse/initializers/disco-toc-main.js

```diff
@@ -240,7 +240,7 @@ export default {
       headings.forEach((heading, subIndex) => {
         if (subIndex > currentIndex && subIndex < nextIndex) {
           let subUl = li.lastChild;
-          if (subUl.tagName !== "ul") {
+          if (subUl.tagName.toLowerCase() !== "ul") {
```

Comment:
From docs:
> For DOM trees which represent HTML documents, the returned tag name is always in the canonical upper-case form.

So it might be simpler to just compare with `"UL"`, but either way is cool